seed/static/seed/js/controllers/inventory_reports_controller.js

```diff
@@ -56,6 +57,10 @@ angular.module('SEED.controller.inventory_reports', []).controller('inventory_re
     $scope.access_level_instance_id = null;
     $scope.users_access_level_instance_id = user_service.get_access_level_instance().id;
 
+    filter_groups_service.get_filter_groups('Property').then((filter_groups) => {
+      $scope.filter_groups = filter_groups;
+    });
```

Comment:
This could be done in `seed.js` too but I dont really think it matters 🤷🏻

Reply:
I've moved this into seed.js - I think that makes good sense.